doc/classes/Animation.xml

```diff
@@ -6,13 +6,24 @@
 	<description>
 		An Animation resource contains data used to animate everything in the engine. Animations are divided into tracks, and each track must be linked to a node. The state of that node can be changed through time, by adding timed keys (events) to the track.
 		[codeblock]
+		[gdscript]
 		# This creates an animation that makes the node "Enemy" move to the right by
 		# 100 pixels in 0.5 seconds.
 		var animation = Animation.new()
 		var track_index = animation.add_track(Animation.TYPE_VALUE)
 		animation.track_set_path(track_index, "Enemy:position:x")
 		animation.track_insert_key(track_index, 0.0, 0)
 		animation.track_insert_key(track_index, 0.5, 100)
+		[/gdscript]
+		[csharp]
+		// This creates an animation that makes the node "Enemy" move to the right by
+		// 100 pixels in 0.5 seconds.
+		var animation = new Animation();
+		int trackIndex = animation.AddTrack(Animation.TYPE_VALUE);
+		animation.TrackSetPath(trackIndex, "Enemy:position:x");
+		animation.TrackInsertKey(trackIndex, 0.0, 0);
+		animation.TrackInsertKey(trackIndex, 0.5, 100);
```

Comment:
This might be good as-is, but double check what type this method accepts. If it's `float`, this needs to be `0.5f`.

Reply:
fixed